### DISCUSSION TITLE
[Feature Request] 类似element的树状导航，可无限层级的那种

### DISCUSSION THREAD

**Discussion by xiezhiyi163** [youzan/vant#11790] [General]
### 这个功能解决了什么问题？

类似element的树状导航，可无限层级的那种，希望可以响应这个功能的实现，并在当前或者未来的版本当中可以进行实现出来，感谢万分！

### 你期望的 API 是什么样子的？

期望使用时候的数据格式
```
[
   text:'xxx',
   id:'1',
   children:[
       {
            text:'xxxb',
            id:'2',
       }
   ]
]
```

**Comment by chenjiahan** [comment]
树形选择在移动端还是比较少见到的，Element UI 的树形选择比较适合桌面端。

可以提供一些参考的 UI 稿看看吗
